courses/serializers.py

```diff
@@ -149,3 +150,47 @@ class Meta:
             'requirements',
             'sections',
         ]
+
+
+class UserProfileSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserProfile
+        fields = [
+            'email',
+            'phone'
+        ]
+
+
+class UserSerializer(serializers.ModelSerializer):
+    profile = UserProfileSerializer(read_only=False)
+
+    def update(self, instance, validated_data):
+        prof, _ = UserProfile.objects.get_or_create(user=instance)
+        prof_data = validated_data.pop('profile')
+        for key in ['username', 'first_name', 'last_name']:
```

Comment:
Wrote this above, but not sure we want to allow updates to these fields since they're coming from Platform (who in turn is getting them from Penn)